### DISCUSSION TITLE
CyclonDX SBOM 1.4 rather than 1.5

### DISCUSSION THREAD

**Discussion by marcelstoer** [aquasecurity/trivy#5233] [Q&A]
### Question

How can I tell trivy to create a CyclonDX SBOM spec 1.4 rather than 1.5? With `cdxgen` for example there's the `--spec-version` flag for that.

`trivy fs --help` does not contain any hints AFAICS.

### Target

Filesystem

### Scanner

None

### Output Format

CycloneDX

### Mode

None

### Operating System

macOS

### Version

```bash
Version: 0.45.1
```

**Comment by DmitriyLewen** [comment]
Hello @marcelstoer 

Trivy doesn't have option to choose SBOM (SPDX or CycloneDX) schema.
We use only latest schemas.

Regards, Dmitriy
